src/Constants.ts

```diff
@@ -6,7 +6,7 @@
 export const CONSTANTS = {
   VersionString: "3.0.0dev",
   isDevBranch: true,
-  VersionNumber: 43,
+  VersionNumber: 44,
```

Comment:
Can you leave this off for now, and we'll just update the changelog first? That can go in immediately, and then bumping the version can be a small change on its own.